openedx/features/course_experience/views/course_home.py

```diff
@@ -35,6 +36,7 @@ def get(self, request, course_id):
         """
         course_key = CourseKey.from_string(course_id)
         course = get_course_with_access(request.user, 'load', course_key, check_if_enrolled=True)
+        add_new_relic_course_key_metrics(course_key)
```

Comment:
I wonder if this could be centralized in ``CourseTabView``. It seems that it would be worth adding these metrics to all course tabs (note that not all course tabs currently subclass from this view, but it is the direction I'd like us to move as we convert tabs to fragments).